src/EditorFeatures/CSharp/BlockCommentEditing/BlockCommentEditingCommandHandler.cs

```diff
@@ -170,7 +171,41 @@ private static bool IsCaretInsideBlockCommentSyntax(SnapshotPoint caretPosition)
             var trivia = syntaxTree.FindTriviaAndAdjustForEndOfFile(caretPosition, CancellationToken.None);
 
             var isBlockComment = trivia.IsKind(SyntaxKind.MultiLineCommentTrivia) || trivia.IsKind(SyntaxKind.MultiLineDocumentationCommentTrivia);
-            return isBlockComment && trivia.FullSpan.Start < caretPosition;
+            if (isBlockComment)
+            {
+                var span = trivia.FullSpan;
+                if (span.Start < caretPosition && caretPosition < span.End)
```

Comment:
Did you mean a test case like `/**$$/`? See https://github.com/diryboy/roslyn/blob/021bac42a0ccdefcdd6a83d247e3ed7274f5220d/src/EditorFeatures/CSharpTest/BlockCommentEditing/BlockCommentEditingTests.cs#L397